src/Game/zCamera.cpp

```diff
@@ -2,37 +2,179 @@
 
 #include <types.h>
 
+#include "zCameraTweak.h"
+#include "../Core/x/xVec3Inlines.h"
+
+enum WallJumpViewState
+{
+	WJVS_DISABLED,
+	WJVS_DISABLING,
+	WJVS_ENABLED,
+	WJVS_ENABLING
+};
+
+struct zFlyKey
+{
+	int32 frame;
+	float32 matrix[12];
+	float32 aperture[2];
+	float32 focal;
+};
+
+// extern int8 buffer[16];
+// extern int8 buffer[16];
+// extern basic_rect screen_bounds;
+// extern basic_rect default_adjust;
+extern float32 zcam_pad_pyaw_scale;
+extern float32 zcam_pad_pitch_scale;
+extern float32 zcam_near_d;
+extern float32 zcam_near_h;
+extern float32 zcam_near_pitch;
+extern float32 zcam_far_d;
+extern float32 zcam_far_h;
+extern float32 zcam_far_pitch;
+extern float32 zcam_wall_d;
+extern float32 zcam_wall_h;
+extern float32 zcam_wall_pitch;
+extern float32 zcam_above_d;
+extern float32 zcam_above_h;
+extern float32 zcam_above_pitch;
+extern float32 zcam_below_d;
+extern float32 zcam_below_h;
+extern float32 zcam_below_pitch;
```

Comment:
The naming convention we've generally been giving floats is:

[name of file]_f_[float value], so for example:

zCamera_f_1_25 or zCamera_float_1_25  for float 1.25 in zCamera. `below_h` doesn't really tell us anything useful, and since the same float value can be re-used in multiple places in code, the names you give them may not make sense when the float is used in other areas. For that reason, we want to keep them named something general.

Reply:
o i just pasted them from the dwarf data, assuming those were the names the devs originally chose.
These are not constants btw, so think they should stay like this (?)

Reply:
Oh, my bad, I thought they were custom names that you chose for the float constants that the compiler generated. If they are variables that the devs created, leave them named like that.